cnames_active.js

```diff
@@ -518,6 +518,7 @@ var cnames_active = {
   "christopher": "marekkobida.github.io/christopher",
   "chronos": "espinielli.github.io/chronos", // noCF? (don´t add this in a new PR)
   "chronosis": "jack-weilage.github.io/chronosis",
+  "circle-ide": "https://circle-ide.github.io/",
```

Comment:
```suggestion
  "circle-ide": "circle-ide.github.io",
```
As the CI error notes, you do not need the https://, nor the trailing slash